routers/repo/view.go

```diff
@@ -195,6 +195,7 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		ctx.Data["ReadmeExist"] = readmeExist
 		if markup.Type(blob.Name()) != "" {
 			ctx.Data["IsRenderedHTML"] = true
+			ctx.Data["IsMarkup"] = true
```

Comment:
Please look for other places with `markup.Type` check. I think it (`ctx.Data["IsMarkup"] = true`) should be added also there (I found https://github.com/JonasFranzDEV/gitea/blob/5aca5a8a3d618630d10272445badcbea5381683f/routers/repo/view.go#L97).

Reply:
Done